runsc/cmd/gofer.go

```diff
@@ -486,11 +486,11 @@ func resolveMounts(conf *config.Config, mounts []specs.Mount, root string) ([]sp
 		}
 		dst, err := resolveSymlinks(root, m.Destination)
 		if err != nil {
-			return nil, fmt.Errorf("resolving symlinks to %q: %v", m.Destination, err)
+			return nil, fmt.Errorf("resolving symlinks to %q: %w", m.Destination, err)
 		}
 		relDst, err := filepath.Rel(root, dst)
 		if err != nil {
-			panic(fmt.Sprintf("%q could not be made relative to %q: %v", dst, root, err))
+			panic(fmt.Sprintf("%q could not be made relative to %q: %w", dst, root, err))
```

Comment:
printf: runsc/cmd/gofer.go:491:10: fmt.Sprintf does not support error-wrapping directive %w